api.go

```diff
@@ -1 +1,77 @@
 package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"requests"
+	"strconv"
+)
+
+var films = []Film.Film{}
+
+type API struct {
+	Core *Core.Core
```

Comment:
core не нужно экспортировать, это детали реализации

Давай что-то типа такого (логгер добавил для будущих использований)

```suggestion
	core *Core.Core
	lg *slog.Logger
```